tests/test_app.py

```diff
@@ -234,7 +234,18 @@ def test_setup_logging_with_envvar():
             encoding=ENCODING,
         )
         logging.getLogger.assert_called_once_with()
-        # ~ assert sys.excepthook == excepthook
+        # clear this to avoid interfering with other tests
+        os.environ.pop("MU_LOG_TO_STDOUT", "")
+
+
```

Comment:
we now setup the exception handler always (not just if logging is turned on) and so this test just confirms that it's set up.